code/datums/mutations.dm

```diff
@@ -530,7 +530,17 @@
 
 /datum/mutation/human/swedish/say_mod(message)
 	if(message)
-		message = replacetext(message,"w","v")
+		message = replacetextEx(message,"W","V")
```

Comment:
replace this and the line below with ```replacetext(message, "w" "v")```
repeat anywhere you've added both the lowercase and uppercase conversions.